geode-book/master_middleman/bookbinder_helpers.rb

```diff
@@ -1,298 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why does Github think that this is gone? If I pull your fork and `git diff origin/develop...origin/feature/GEODE-3416`, I don't see any change to this file.